mozregression/launchers.py

```diff
@@ -289,13 +289,13 @@ class FennecLauncher(Launcher):
 
     @classmethod
     def check_is_runnable(cls):
-        # ADBHost().devices() seems to raise OSError when adb is not
-        # installed and in PATH. TODO: maybe fix this in mozdevice.
+        # ADBHost().devices() seems to raise ADBError
+        # when adb is not executable or not in PATH.
         try:
             devices = ADBHost().devices()
-        except OSError:
+        except ADBError:
             raise LauncherNotRunnable("adb (Android Debug Bridge) is not"
-                                      " installed or not in the PATH.")
+                                      " executable or not in the PATH.")
```

Comment:
Oh, if you want, we can chat on irc, I am parkouss in the \#ateam channel. You can connect with mibbit for example:
https://wiki.mozilla.org/IRC#Mibbit